packages/govuk-frontend/src/govuk/settings/_typography-responsive.scss

```diff
@@ -140,6 +140,26 @@ $govuk-typography-scale: (
     )
   ),
   14: (
+    null: (
+      font-size: 12px,
+      line-height: 15px
+    ),
+    tablet: (
+      font-size: 14px,
+      line-height: 20px
+    ),
+    print: (
+      font-size: 12pt,
+      line-height: 1.2
+    ),
+
+    deprecation: (
+      key: "govuk-typography-scale-14",
+      message: "14 on the typography scale is deprecated and will be removed as " +
+        "a possible option in the next major version."
+    )
+  ),
+  _14: (
```

Comment:
Wondering whether a duplicate `_14` key/map pair is 100% necessary?

For example, we could automatically remove the underscores:

```scss
// Remove underscore from deprecated typography scale keys
@if type-of($size) == "string" and str-slice($size, 1, 1) == "_" {

  // Remove underscore, but still a string
  $size: str-slice($size, 2);
}
```

But it seemed to complicated to convert `_14` (string) into `14` (number) for the lookup to work:

```scss
// The $size argument type must match the map key
$font-map: map-get($govuk-typography-scale, $size);
```

Reply:
I had a go at implementing it but can't get past the variable type at the end without it getting just a bit too complicated. Do you have some other ideas for removing the extra scale value? Otherwise I think this is probably the simplest way to get the thing done, even if it isn't the best.

Reply:
Nah, leave it the . If the keys were already strings we'd be fine